ruby/santa.rb

```diff
@@ -1,6 +1,9 @@
 #Release 0 
 
 class Santa 
+	attr_reader :age, :ethnicity
```

Comment:
you have a reader for age but nothing setting it.  it is my opinion that santa's age should be set upon initialization of an instance of the santa class